h2o-py/h2o/assembly.py

```diff
@@ -55,11 +101,21 @@ def names(self):
 
 
     def to_pojo(self, pojo_name="", path="", get_jar=True):
+        """
+        Convert the munging operations performed on H2OFrame into a POJO.
+
+        :param pojo_name (str): Name of POJO
+        :param path (str): path of POJO.
+        :param get_jar (bool): Whether to also download the h2o-genmodel.jar file needed to compile the POJO
+        :return: None
+        """
         assert_is_type(pojo_name, str)
         assert_is_type(path, str)
         assert_is_type(get_jar, bool)
         if pojo_name == "":
             pojo_name = "AssemblyPOJO_" + str(uuid.uuid4())
+
+        if pojo_name == "": pojo_name = "AssemblyPOJO_" + str(uuid.uuid4())
         java = h2o.api("GET /99/Assembly.java/%s/%s" % (self.id, pojo_name))
         file_path = path + "/" + pojo_name + ".java"
         if path == "":
```

Comment:
in the lines 127-131 below, replace all that nonsense with a single call to h2o.api(). Check `download_mojo` in `model_base.py` for an example